asyncapi/implementation/src/main/java/org/iris_events/asyncapi/parsers/MessageIsRpcParser.java

```diff
@@ -0,0 +1,18 @@
+package org.iris_events.asyncapi.parsers;
+
+import org.iris_events.annotations.Message;
+import org.jboss.jandex.AnnotationInstance;
+import org.jboss.jandex.IndexView;
+
+public class MessageIsRpcParser {
```

Comment:
IS_RPC is not really needed, could use the "responseEventName" paramter for both.